specs/xaml-backdrop-api.md

```diff
@@ -0,0 +1,316 @@
+
+This is useful if you're using a custom `SystemBackdropConfiguration`.
+The following example shows a `MicaBackdrop` similar to the previous example,
+but always forces the theme to be light.
+
+```cs
+protected override void OnSystemDefaultBackdropConfigurationChanged(SystemDefaultBackdropChangedEventArgs e)
+{
+    SetControllerConfig(e.ConnectedTarget, e.AssociatedRoot);
+}
+
+void SetControllerConfig(ICompositionSupportsSystemBackdrop connectedTarget, XamlRoot xamlRoot)
+{
+    var config = GetDefaultSystemBackdropConfiguration(connectedTarget, xamlRoot);
+    config.Theme = SystemBackdropTheme.Light;
+    _micaController.SetSystemBackdropConfiguration(config);
+}
+
+```
+
+
+
+## Other `SystemBackdrop` members
+
+| Member | Description |
+| - | - |
+| `OnTargetConnected` virtual method | Called when this object is attached to an object, for example when set on `Window.SystemBackdrop` |
+| `OnTargetDisconnected` virtual method | Called when this object is cleared from an object |
+
+
+
+## SystemDefaultBackdropChangedEventArgs class
+
+Event args for the `OnSystemDefaultBackdropConfigurationChanged` virtual method.
+Provides an updated `ICompositionSupportsSystemBackdrop` and `XamlRoot`.
+
+
+
+# API Details
+
+```c# (but really MIDL3)
+namespace Microsoft.UI.Xaml.Media
+{
+  [contract(Microsoft.UI.Xaml.WinUIContract, 4)]
+  [webhosthidden]
+  unsealed runtimeclass MicaBackdrop : Microsoft.UI.Xaml.Media.SystemBackdrop
+  {
+      MicaBackdrop();
+
+      Microsoft.UI.Composition.SystemBackdrops.MicaKind Kind { get; set; };
+
+      static Microsoft.UI.Xaml.DependencyProperty KindProperty { get; };
+  }
+
+  [contract(Microsoft.UI.Xaml.WinUIContract, 4)]
+  [webhosthidden]
+  unsealed runtimeclass DesktopAcrylicBackdrop : Microsoft.UI.Xaml.Media.SystemBackdrop
+  {
+      DesktopAcrylicBackdrop();
+  }
+
+  [contract(Microsoft.UI.Xaml.WinUIContract, 4)]
+  [webhosthidden]
+  unsealed runtimeclass SystemBackdrop
+      : Microsoft.UI.Xaml.DependencyObject
+  {
+      protected SystemBackdrop();
+      
+      overridable void OnTargetConnected(
+        Microsoft.UI.Composition.ICompositionSupportsSystemBackdrop connectedTarget,
+        Microsoft.UI.Xaml.XamlRoot xamlRoot);
+      overridable void OnTargetDisconnected(
+        Microsoft.UI.Composition.ICompositionSupportsSystemBackdrop disconnectedTarget);
+
+      protected Microsoft.UI.Composition.SystemBackdrops.SystemBackdropConfiguration 
+        GetDefaultSystemBackdropConfiguration(
+          Microsoft.UI.Composition.ICompositionSupportsSystemBackdrop target, 
+          Microsoft.UI.Xaml.XamlRoot xamlRoot);
+
+        protected virtual void OnSystemDefaultBackdropConfigurationChanged(
+            SystemDefaultBackdropChangedEventArgs e)
+  };
+
+  [contract(Microsoft.UI.Xaml.WinUIContract, 4)]
+  [webhosthidden]
+  unsealed runtimeclass SystemBackdropChangedEventArgs
```

Comment:
>SystemBackdropChangedEventArgs [](http://example.com/codeflow?start=24&length=30)

Naming of arg is inconsistent with naming of function using it. Suggest one of the following:
"SystemBackdropConfigurationChangedEventArgs"
or 
"DefaultSystemBackdropConfigurationChangedEventArgs"
or 
"DefaultConfigurationChangedEventArgs" (<-- matching my earlier comment)

Reply:
Args type not present in final design